modules/cnv-storage-wizard-fields-web.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/TBD
+
+[id="cnv-storage-wizard-fields-web_{context}"]
+== Storage fields
+|===
+|Name | Description
```

Comment:
Do you edit these values? Is a table the best place to present this information?

Consider adding a column for default values, if applicable, and a column to describe the valid values.

Reply:
n/a. I didn't create this module. Resolving all comments that are unrelated to my changes